spread.yaml

```diff
@@ -57,6 +57,24 @@ backends:
                 username: ubuntu
                 password: ubuntu
 
+    external-ubuntu-core:
+        type: adhoc
+        environment:
+            SPREAD_ADHOC_UC_ADDRESS: "$(HOST: echo ${SPREAD_ADHOC_UC_ADDRESS:-localhost:8022})"
+        allocate: |
+            ADDRESS $SPREAD_ADHOC_UC_ADDRESS
+        systems:
+            # FIXME: remove the ubuntu-core-16.04-64 dummy system once the runner handles tasks without systems assigned
+            - ubuntu-core-16.04-64:
+                username: test
+                password: ubuntu
+            - ubuntu-core-16-device-amd64:
```

Comment:
Ok, I've added the changes, instead of using an additional environment variable specific to the unmodified ubuntu-core system (the one in the external backend), which gives an unbounded variable error in the customized system (linode backend) where the variable is not present, I've used `$SPREAD_BACKEND` for determining if a test should be skipped or not. At the moment this just affects two tests, ack and ubuntu-core-device-reg. Please, have a look.

I've also taken the opportunity to review the tests that were disabled for ubuntu-core-16-64 and that  work well on the boards, I've made the required adjustments and now they run in both backends, linode and external.